lib/widgets/appBar.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
```

Comment:
Also, the name of file should follow dart name convention.
https://dart.dev/effective-dart/style